src/main/resources/assets/js/app/inputtype/ui/selector/ContentComboBox.ts

```diff
@@ -67,10 +74,54 @@ export class ContentComboBox<ITEM_TYPE extends ContentTreeSelectorItem>
         }
 
         this.showAfterReload = false;
-
         this.optionsFactory = new OptionsFactory<ITEM_TYPE>(this.getLoader(), builder.optionDataHelper);
     }
 
+    private createStatusColumn() {
+        this.statusColumn = new GridColumnBuilder()
+            .setId('status')
+            .setName('Status')
+            .setField('displayValue')
+            .setFormatter(ContentRowFormatter.statusSelectorFormatter)
+            .setCssClass('status')
+            .setBoundaryWidth(75, 75)
+            .build();
+    }
+
+    protected initListeners() {
+        const debouncedHandler = AppHelper.debounce(this.handleAvailableSizeChanged.bind(this), 300);
+        ResponsiveManager.onAvailableSizeChanged(this, debouncedHandler);
+    }
+
+    private handleAvailableSizeChanged() {
+        const columns: GridColumn<any>[] = this.getColumnsWithoutCheckbox();
+        const width: number = this.getEl().getWidth();
+
+        if (ResponsiveRanges._360_540.isFitOrSmaller(width)) {
+            if (this.isStatusColumnShown()) {
+                const newColumns: GridColumn<any>[] = columns.filter((column: GridColumn<any>) => column.id !== 'status') ;
```

Comment:
Compare with `this.statusColumn.getId()`.

Reply:
Won't work: ssome of slickgrid objects (checkbox in our case) don't have `getId()` method